src/components/geocoder-panel.js

```diff
@@ -51,63 +51,13 @@ const ICON_LAYER = {
   }
 };
 
-const GeocoderPanelContent = styled.div`
-  background-color: ${props => props.theme.panelBackground};
-  padding: 0.5em 1em;
+const StyledGeocoderPanel = styled.div`
   position: absolute;
-  top: 20px;
-  left: 50%;
-  margin-left: -20em;
-  width: 40em;
-  box-sizing: border-box;
-`;
-
-const GeocoderWrapper = styled.div`
-  color: ${props => props.theme.textColor};
-  font-size: ${props => props.theme.fontSize};
-
-  .react-geocoder {
-    position: relative;
-  }
-
-  .react-geocoder input {
-    ${props => props.theme.secondaryInput};
-  }
-
-  .react-geocoder-results {
-    background-color: ${props => props.theme.panelBackground};
-    position: absolute;
-    width: 43.5em;
-  }
-
-  .react-geocoder-item {
-    ${props => props.theme.dropdownListItem};
-    ${props => props.theme.textTruncate};
-  }
-
-  .remove-layer {
-    background: transparent;
-    border: none;
-    bottom: 28px;
-    color: ${props => props.theme.textColor};
-    cursor: pointer;
-    display: inline;
-    font-size: 16px;
-    padding: 2px 8px;
-    position: absolute;
-    right: 16px;
-
-    :hover,
-    :focus,
-    :active {
-      background: transparent !important;
-      border: none;
-      box-shadow: 0;
-      color: ${props => props.theme.textColor};
-      opacity: 0.6;
-      outline: none;
-    }
-  }
+  top: ${props => props.theme.geocoderTop}px;
```

Comment:
Same with **'px'**
Can you imagine if you will have to move from **px** to **em** someday?

Reply:
it's easy to do math without `px`, e.g. XXmargin = props.theme.geocoderTop + props.theme.sidepanelPadding